src/commands/context/spec/context.spec.server.ts

```diff
@@ -93,11 +120,72 @@ describe('sasjs context', () => {
       },
       60 * 1000
     )
+
+    it(
+      'should return an error if trying to create compute context that already exists',
+      async () => {
+        testContextConfigFile = sanitizeFileName(contexts[0].name) + '.json'
+        testContextConfigPath = path.join(process.cwd(), testContextConfigFile)
+
+        let contextConfig = await readFile(testContextConfigPath)
+
+        testContextConfig = JSON.parse(contextConfig)
+        testContextConfig = {
+          ...testContextConfig,
+          launchContext: {
+            contextName: 'CLI Unit Tests launcher context'
+          }
+        }
+
+        if (!testContextConfig.attributes) {
+          testContextConfig.attributes = { runServerAs: 'cas' }
+        }
+
+        contextConfig = JSON.stringify(testContextConfig, null, 2)
+
+        await createFile(testContextConfigPath, contextConfig)
+
+        const command = new Command(
+          `context create -s ${testContextConfigFile} -t targetName`
+        )
+
+        await expect(processContext(command)).resolves.toEqual(
+          new Error(
+            `Compute context '${testContextConfig.name}' already exists.`
+          )
+        )
+      },
+      60 * 1000
+    )
   })
 
   describe('edit', () => {
     it(
-      'should edit a compute context',
+      'should return an error if trying to edit default compute context',
+      async () => {
+        testContextConfig.description += '_updated'
+
+        setDefaultContextName()
+
+        const contextConfig = JSON.stringify(testContextConfig, null, 2)
+
+        await createFile(testContextConfigPath, contextConfig)
+
+        const command = new Command(
+          `context edit ${testContextConfig.name} -s ${testContextConfigFile} -t ${targetName}`
+        )
+
+        await expect(processContext(command)).resolves.toEqual(
+          defaultContextError('Editing')
+        )
+
+        restoreTestContextName()
+      },
+      60 * 1000
```

Comment:
It's best to use a `done` callback here and avoid specifying an explicit timeout, since that is already set in the `jest.config.js` file.